### ISSUE TITLE
Crash when trying to open any song

### ISSUE THREAD

**Issue by Senjosei** [zyrouge/symphony#7]
Crash when trying to open any songs
Android 13

```FATAL EXCEPTION: main
Process: io.github.zyrouge.symphony, PID: 6581
java.lang.IllegalArgumentException: Cannot round NaN value.
	at a0.a.d0(Unknown Source:15)
	at q.u.e(Unknown Source:24)
	at j1.c.e(Unknown Source:14)
	at j1.t.e(Unknown Source:10)
	at j1.b0.y(Unknown Source:8)
	at n0.h$a.a(Unknown Source:63)
	at n0.z.y(Unknown Source:6)
	at androidx.activity.m.P0(Unknown Source:43)
	at j1.y0.a(Unknown Source:74)
	at j1.z$b.c1(Unknown Source:171)
	at j1.z$b.e(Unknown Source:168)
	at q.i.b(Unknown Source:217)
	at j1.n.e(Unknown Source:43)
	at q.u.e(Unknown Source:99)
	at j1.c.e(Unknown Source:14)
	at j1.t.e(Unknown Source:10)
	at q.s1.e(Unknown Source:121)
	at j1.c.e(Unknown Source:14)
	at j1.t.e(Unknown Source:10)
	at j1.b0.y(Unknown Source:8)
	at n0.h$a.a(Unknown Source:63)
	at n0.z.y(Unknown Source:6)
	at androidx.activity.m.P0(Unknown Source:43)
	at j1.y0.a(Unknown Source:74)
	at j1.z$b.c1(Unknown Source:171)
	at j1.z$b.e(Unknown Source:168)
	at q.i1.b(Unknown Source:177)
	at j1.n.e(Unknown Source:43)
	at j1.b0.y(Unknown Source:8)
	at n0.h$a.a(Unknown Source:63)
	at n0.z.y(Unknown Source:6)
	at androidx.activity.m.P0(Unknown Source:43)
	at j1.y0.a(Unknown Source:74)
	at j1.z$b.c1(Unknown Source:171)
	at j1.z$b.e(Unknown Source:168)
	at l.l.b(Unknown Source:37)
	at j1.n.e(Unknown Source:43)
	at u0.k.e(Unknown Source:5)
	at j1.c.e(Unknown Source:14)
	at j1.t.e(Unknown Source:10)
	at l.v0.e(Unknown Source:5)
	at j1.c.e(Unknown Source:14)
	at j1.t.e(Unknown Source:10)
	at j1.b0.y(Unknown Source:8)
	at n0.h$a.a(Unknown Source:63)
	at n0.z.y(Unknown Source:6)
	at androidx.activity.m.P0(Unknown Source:43)
	at j1.y0.a(Unknown Source:74)
	at j1.z$b.c1(Unknown Source:171)
	at j1.z$b.e(Unknown Source:168)
	at q.i1.b(Unknown Source:177)
	at j1.n.e(Unknown Source:43)
	at j1.b0.y(Unknown Source:8)
	at n0.h$a.a(Unknown Source:63)
	at n0.z.y(Unknown Source:6)
	at androidx.activity.m.P0(Unknown Source:43)
	at j1.y0.a(Unknown Source:74)
	at j1.z$b.c1(Unknown Source:171)
	at j1.v.O(Unknown Source:16)
	at j1.e0.d(Unknown Source:24)
	at j1.e0.k(Unknown Source:130)
	at j1.e0.g(Unknown Source:68)
	at androidx.compose.ui.platform.AndroidComposeView.a(Unknown Source:13)
	at androidx.compose.ui.platform.AndroidComposeView.dispatchDraw(Unknown Source:20)
	at android.view.View.draw(View.java:23198)
	at android.view.View.updateDisplayListIfDirty(View.java:22062)
	at android.view.ViewGroup.recreateChildDisplayList(ViewGroup.java:4513)
	at android.view.ViewGroup.dispatchGetDisplayList(ViewGroup.java:4486)
	at android.view.View.updateDisplayListIfDirty(View.java:22018)
	at android.view.ViewGroup.recreateChildDisplayList(ViewGroup.java:4513)
	at android.view.ViewGroup.dispatchGetDisplayList(ViewGroup.java:4486)
	at android.view.View.updateDisplayListIfDirty(View.java:22018)
	at android.view.ViewGroup.recreateChildDisplayList(ViewGroup.java:4513)
	at android.view.ViewGroup.dispatchGetDisplayList(ViewGroup.java:4486)
	at android.view.View.updateDisplayListIfDirty(View.java:22018)
	at android.view.ViewGroup.recreateChildDisplayList(ViewGroup.java:4513)
	at android.view.ViewGroup.dispatchGetDisplayList(ViewGroup.java:4486)
	at android.view.View.updateDisplayListIfDirty(View.java:22018)
	at android.view.ThreadedRenderer.updateViewTreeDisplayList(ThreadedRenderer.java:682)
	at android.view.ThreadedRenderer.updateRootDisplayList(ThreadedRenderer.java:688)
	at android.view.ThreadedRenderer.draw(ThreadedRenderer.java:786)
	at android.view.ViewRootImpl.draw(ViewRootImpl.java:4579)
	at android.view.ViewRootImpl.performDraw(ViewRootImpl.java:4290)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:3517)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:2286)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:8959)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1231)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1239)
	at android.view.Choreographer.doCallbacks(Choreographer.java:899)
	at android.view.Choreographer.doFrame(Choreographer.java:832)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1214)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7900)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
```

**Comment by zyrouge** [comment]
Could you try clearing cache and storage? Also, mind attaching a screen record of that happening?

**Comment by zyrouge** [comment]
Ok, nevermind. Seems to occur in release versions.

**Comment by zyrouge** [comment]
Fixed as of v2022.11.38!
